tests/suite/test_suite.py

```diff
@@ -86,6 +87,9 @@
     e2e_logging.test_view_history,
     e2e_logging.test_tx_statuses,
     e2e_logging.test_random_receipts,
+    # e2e_localcommithandler
+    e2e_localcommithandler.test_safe,
+    e2e_localcommithandler.test_unsafe,
```

Comment:
I think this is OK not to include in the full test suite as this is a new stateless (i.e. does not impact recovery, ledger, etc.).